src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -301,7 +304,11 @@ internal void Connect(ServerInfo serverInfo, SqlInternalConnectionTds connHandle
             _connHandler = connHandler;
             _loginWithFailover = withFailover;
 
+#if MANAGED_SNI
+            UInt32 sniStatus = SNILoadHandle.SingletonInstance.Status;
```

Comment:
Can we just name the properties the same so that we don't need the #if: we just end up including the right file and the right things happen?  Or is there some reason we can't do that?